test/lib/base-controller.js

```diff
@@ -329,7 +329,67 @@ describe('lib/base-controller', function () {
             }];
             controller.options.continueOnEdit = false;
             req.params.action = 'edit';
-            controller.getNextStep(req).should.contain('/confirm');
+            controller.getNextStep(req).should.equal('/confirm');
+          });
+
+          it('should return /a-base-url/confirm if baseUrl is set', function () {
+            getStub.returns(['/target-page']);
+            req.form.values['example-radio'] = 'superman';
+            controller.options.forks = [{
+              target: '/target-page',
+              condition: function (request) {
+                return request.form.values['example-radio'] === 'superman';
+              }
+            }];
+            controller.options.continueOnEdit = false;
+            req.params.action = 'edit';
+            req.baseUrl = '/a-base-url';
+            controller.getNextStep(req).should.equal('/a-base-url/confirm');
+          });
+
+          it('should return /a-base-url/target-page if completed check falsy', function () {
+            getStub.returns(['/a-base-url/target-page']);
+            req.form.values['example-radio'] = 'superman';
+            controller.options.forks = [{
+              target: '/target-page',
+              condition: function (request) {
+                return request.form.values['example-radio'] === 'superman';
+              }
+            }];
+            controller.options.continueOnEdit = false;
+            req.params.action = 'edit';
+            req.baseUrl = '/a-base-url';
+            controller.getNextStep(req).should.equal('/a-base-url/target-page');
+          });
+
+          it('should append "edit" to the path if baseUrl is set and continueOnEdit is false', function () {
+            getStub.returns(['/target-page']);
+            req.form.values['example-radio'] = 'superman';
+            controller.options.forks = [{
+              target: '/target-page',
+              condition: function (request) {
+                return request.form.values['example-radio'] === 'superman';
+              }
+            }];
+            controller.options.continueOnEdit = true;
+            req.params.action = 'edit';
+            req.baseUrl = '/a-base-url';
+            controller.getNextStep(req).should.equal('/a-base-url/target-page/edit');
+          });
+
+          it('should not append "edit" to the path if contains function returns falsy', function () {
+            getStub.returns(['/a-base-url/target-page']);
```

Comment:
👍  Yeah on second thoughts, they aren't testing anything new. The bottom one I added has it covered - removing now